packages/docs/docs/00-introduction.mdx

```diff
@@ -39,24 +39,6 @@ import Form from '@rjsf/core';
 
 Our latest version requires React 16+. You can also install `react-jsonschema-form` (the 1.x version) which works with React 15+.
 
-### As a script served from a CDN
```

Comment:
`core.cjs.production.min.js` is to be no longer correct after the changes (new link should be to `index.js` and it is also not minified)

However, for both urls - it is a commonjs bundle, which will not work in browser environment because it will try to "require" all the dependencies (react, lodash, etc…) 

This can be changes to umd bundle, but that also would not be easy - requiring bunch of script tags with all the dependencies. And so, I believe, it is better to just remove this section entirely to avoid confusion and also to consider umd bundle as deprecated and to remove that in next major version(in case anyone still using it)